src/project_manager/CMakeLists.txt

```diff
@@ -3,11 +3,19 @@ set(CMAKE_AUTORCC ON)
 set(CMAKE_AUTOUIC ON)
 set(CMAKE_CXX_STANDARD 17)
 
+set(YAMLCPP_USE_STATIC_LIBS TRUE)
+
 find_package(QtCreator COMPONENTS Core REQUIRED)
 find_package(QT NAMES Qt6 Qt5 COMPONENTS Widgets REQUIRED)
 find_package(yaml-cpp REQUIRED)
 find_package(qtermwidget5 REQUIRED)
 
+message("YAMLCPP_FOUND:" ${YAMLCPP_FOUND})
+message("YAMLCPP_LIBRARY:" ${YAMLCPP_LIBRARY})
+message("YAMLCPP_LIBRARY_DIR:" ${YAMLCPP_LIBRARY_DIR})
+message("YAMLCPP_USE_STATIC_LIBS:" ${YAMLCPP_USE_STATIC_LIBS})
+message("YAMLCPP_STATIC:" ${YAMLCPP_STATIC})
```

Comment:
Do you want to leave these here or should they be removed?

Reply:
They should be removed. I added them temporarily to debug a linking issue on macOS.